src/site/_includes/components/SignPosts.js

```diff
@@ -15,28 +15,28 @@
  */
 
 const {html} = require('common-tags');
-const stripLanguage = require('../../_filters/strip-language');
+const paths = require('../../_data/paths');
+const postToPaths = require('../../_data/postToPaths');
 
-module.exports = ({postToCollections, url}) => {
-  // When @devnook lands her localization work we might need to update this file.
-  const strippedUrl = stripLanguage(url).replace(/\//g, '');
-  const collections = postToCollections[strippedUrl] || [];
-  const aTags = collections.map((collection) => {
+module.exports = (slug, x) => {
+  const postPaths = postToPaths[slug];
+  if (!postPaths) {
+    return '';
+  }
+  const aTags = postPaths.map((pathName) => {
     return html`
-      <a class="w-post-signpost__link" href="${collection.href}"
-        >${collection.title}</a
+      <a class="w-post-signpost__link" href="${paths[pathName].href}"
```

Comment:
this doesn't seem to be working. try the signpost element on this page:
https://deploy-preview-3155--web-dev-staging.netlify.app/document-title/

both links go back to the page itself, not the paths

Reply:
Forgot to update property name (s/href/slug)